### PR TITLE
Stable Pools and view for BPT balances

### PR DESCRIPTION
**Pull request by markusbkoch**

### CODE REVIEW

**Comment by markusbkoch** [comment]
ready for review @medvedev1088

**Comment by markusbkoch** [comment]
I meant to do the stkABPT in a separate PR, but I guess this is fine too

File: dags/resources/stages/parse/table_definitions/balancer/V1_ stkABPT_event_Transfer.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/balancer/V1_ stkABPT_event_Transfer.json:1]: @markusbkoch It looks like there is a space in the file name?

Comment (markusbkoch) [reply dags/resources/stages/parse/table_definitions/balancer/V1_ stkABPT_event_Transfer.json]: Thank you, fixed

**Pull request merged**